secretmanager/create_secret_with_tags.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package secretmanager
+
+// [START secretmanager_create_secret_with_labels]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	secretmanager "cloud.google.com/go/secretmanager/apiv1"
+	"cloud.google.com/go/secretmanager/apiv1/secretmanagerpb"
+)
+
+// createSecretWithTags creates a new secret with the given name and labels.
+func createSecretWithTags(w io.Writer, parent, id string, tagKey string, tagValue string) error {
```

Comment:
nit: declare each parameter with type or declare all strings and then declare type. 

The `parent` parameter doesn't have a type annotation, so it picks up the type from `id`. The remaining parameters all have individual type annotations. 

Suggestion:

```go
func createSecretWithTags(w io.Writer, parent, id, tagKey, tagValue string) error {
 // ...
```